PreSignUpLambdaHook/index.mjs

```diff
@@ -1,35 +1,26 @@
-﻿import pg from "pg";
-import fs from "fs";
+﻿import pg from 'pg';
+import fs from 'fs/promises';
 
-export const handler = async (event, context, callback) => {
+const checkEmailAddress = async (emailAddress) => {
     const client = new pg.Client({
         ssl: {
-            rejectUnauthorized: false,
-            ca: fs.readFileSync("certs/eu-west-2-bundle.pem").toString(),
+            ca: await fs.readFile('certs/eu-west-2-bundle.pem', 'utf-8'),
```

Comment:
If you provide a text encoding (which you always should!) then it'll return a string, plus we can use the promise version of `fs`